mypy_django_plugin/main.py

```diff
@@ -334,11 +334,13 @@ def get_attribute_hook(self, fullname: str) -> Optional[Callable[[AttributeConte
             return partial(request.set_auth_user_model_as_type_for_request_user, django_context=self.django_context)
         if info and info.has_base(fullnames.ABSTRACT_USER_MODEL_FULLNAME) and attr_name in ("is_staff", "is_active"):
             return partial(set_auth_user_model_boolean_fields, django_context=self.django_context)
-        if (
+
+        is_dynamic_manager_class = (
```

Comment:
Let's refactor other conditions as well! 
At least in this hook.

Reply:
Yeah, sounds like a good idea

Reply:
IMO having all statements broken out to individual variables became a bit messy. I went with comments instead, WDYT?